src/components/App.js

```diff
@@ -38,6 +40,7 @@ function App() {
   const getUser = async (id) => {
     const user = await getDoc(doc(db, "user", id));
     if (user.exists()) {
+   
       return user.data();
```

Comment:
Prefer using the 'Early return' tip mentioned before.
Ref: https://youtu.be/cfeKek8p9us